code/modules/reagents/chemistry/reagents/alcohol_reagents.dm

```diff
@@ -225,9 +225,9 @@ All effects don't start immediately, but rather get worse over time; the rate is
 /datum/reagent/consumable/ethanol/vodka
 	name = "Vodka"
 	description = "Number one drink AND fueling choice for Russians worldwide."
-	color = "#0064C8" // rgb: 0, 100, 200
+	color = "#AAAAAA77" // rgb: 170, 170, 170, 77 (alpha) (clear like water)
 	boozepwr = 65
-	taste_description = "grain alcohol"
+	taste_description = "vodka"
```

Comment:
Consider adding a hydration_factor, sake, ale and beer use 10. What do you think would be appropriate for vodka? Default is 5.

Reply:
Ten as all others. Sake and vodka are pretty similar, and if we're being awfully technical, I'd say most drinks make you a little thirsty, so if anything, their hydration should be lowered to default.